src/components/Video.js

```diff
@@ -154,6 +154,7 @@ export default class Video extends Component {
         .catch(error => {})
         .then(() => {});
     }
+    return promise;
```

Comment:
This whole code is:

```
  play() {
    const promise = this.video.play();
    if (promise !== undefined) {
      promise
        .catch(error => {})
        .then(() => {});
    }
    return promise;
  }
```

It seems to be working but I think I should double check how that promise is working. Never had a promise that has a then and catch, returns itself, and does another then and catch.